### ISSUE TITLE
sql server column comment not work

### ISSUE THREAD

**Issue by Fire-Dragons** [sqlalchemy/sqlalchemy#8074]
### Describe the bug

sql server column comment not work

### To Reproduce

```python
from sqlalchemy import create_engine, String, DateTime, Column
from sqlalchemy.ext.declarative import declarative_base
from sqlalchemy import Column, String, Integer, Text
from sqlalchemy import ForeignKey, Sequence, MetaData, Table
from sqlalchemy.orm import relationship, sessionmaker
from sqlalchemy.ext.horizontal_shard import ShardedSession, ShardedQuery
from sqlalchemy import Enum
import os
import uuid
import datetime
import enum


# 初始化数据库链接pool_pre_ping=True, echo=True)
engine = create_engine('mssql+pymssql://user:pwd@host:1433/db?charset=utf8', pool_recycle=600, pool_pre_ping=True, echo=True)
# 创建对象的基类
md = MetaData(bind=engine)  # 引用MetaData

Base = declarative_base(metadata = md)

def global_id():
    return str(uuid.uuid4()).replace('-','')

class BaseStatusEnum(enum.Enum):
    available = 1
    deleted = 2

class user(Base):
    # 表的名字
    __tablename__ = 'user'
    id = Column(String(32), nullable=False, primary_key=True, default=global_id, comment="主键")
    name = Column(String(32), comment="名字")

Base.metadata.create_all(engine)
```

### Error

```
2022-05-30 17:08:38,466 INFO sqlalchemy.engine.Engine select @@version
2022-05-30 17:08:38,466 INFO sqlalchemy.engine.Engine [raw sql] {}
2022-05-30 17:08:38,468 INFO sqlalchemy.engine.Engine SELECT schema_name()
2022-05-30 17:08:38,468 INFO sqlalchemy.engine.Engine [generated in 0.00009s] {}
2022-05-30 17:08:38,471 INFO sqlalchemy.engine.Engine SELECT CAST('test max support' AS NVARCHAR(max))
2022-05-30 17:08:38,471 INFO sqlalchemy.engine.Engine [generated in 0.00010s] {}
2022-05-30 17:08:38,476 INFO sqlalchemy.engine.Engine BEGIN (implicit)
2022-05-30 17:08:38,480 INFO sqlalchemy.engine.Engine SELECT [INFORMATION_SCHEMA].[TABLES].[TABLE_NAME]
FROM [INFORMATION_SCHEMA].[TABLES]
WHERE [INFORMATION_SCHEMA].[TABLES].[TABLE_TYPE] = CAST(%(table_type_1)s AS NVARCHAR(max)) AND [INFORMATION_SCHEMA].[TABLES].[TABLE_NAME] = CAST(%(table_name_1)s AS NVARCHAR(max)) AND [INFORMATION_SCHEMA].[TABLES].[TABLE_SCHEMA] = CAST(%(table_schema_1)s AS NVARCHAR(max))
2022-05-30 17:08:38,480 INFO sqlalchemy.engine.Engine [generated in 0.00015s] {'table_type_1': 'BASE TABLE', 'table_name_1': 'user', 'table_schema_1': 'dbo'}
2022-05-30 17:08:38,482 INFO sqlalchemy.engine.Engine
CREATE TABLE [user] (
	id VARCHAR(32) NOT NULL,
	name VARCHAR(32) NULL,
	PRIMARY KEY (id)
)


2022-05-30 17:08:38,482 INFO sqlalchemy.engine.Engine [no key 0.00017s] {}
2022-05-30 17:08:38,512 INFO sqlalchemy.engine.Engine COMMIT
``` 

### Versions

- OS:mac 12.4
- Python:3.9.7
- SQLAlchemy:1.4.36
- Database:sql server
- DBAPI (eg: psycopg, cx_oracle, mysqlclient):pymssql

**Comment by Fire-Dragons** [comment]
if database column have comment but md.reflect(bind=engine, views=True) and md.tables['user'].columns.name.comment is None

**Comment by zzzeek** [comment]
not implemented, see #7844
